src/main/core/environment.ts

```diff
@@ -5,6 +5,7 @@
  * LICENSE file in the root directory of this source tree.
  */
 import { isCI } from 'ci-info'
+import isInsideContainer from 'is-inside-container'
```

Comment:
SonarCloud is flagging this line as uncovered code? @IgnacioBecerra

Reply:
Unsure why this is marked as uncovered, I'm testing the function and condition itself. I'll keep looking into it..